ModuleTypes/SubModuleChild.cs

```diff
@@ -12,9 +12,9 @@ namespace ConnectWise.Http.ModuleTypes
     /// </summary>
     public abstract class SubModuleChild
     {
-        private string module;
-        private string endpoint;
-        private string child;
+        private readonly string module;
+        private readonly string endpoint;
+        private readonly string child;

```

Comment:
If we're going to make a minor little change like this I would prefer that we either do it everywhere or not at all, just for consistency.